pkgs/development/python-modules/yubico-client/default.nix

```diff
@@ -0,0 +1,23 @@
+{ stdenv, lib, buildPythonPackage, isPyPy, fetchPypi
+, requests }:
+
+buildPythonPackage rec {
+  pname = "yubico-client";
+  version = "1.10.0";
+
+  disabled = isPyPy;
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "0skkmrpvpb1pwyqjf3lh9vq46xagvwdx9kagpdbba2v5dgrk34d1";
+  };
+
+  propagatedBuildInputs = [ requests ];
+
+  doCheck = false;
```

Comment:
We'd like to enable tests. Why have they been disabled?

Reply:
i got
```
ModuleNotFoundError: No module named 'test_utils'
```

Reply:
i guess i can do anther fetchfromgit